docs/subnets/how-to-transfer-funds.md

```diff
@@ -0,0 +1,332 @@
+
+## Example: Sending All Funds From One Ledger to Another
+
+- Source address: ledger A, index 2 (the web wallet shows 4.5 AVAX for this ledger)
+- Target address: ledger B, index 0 (the web wallet shows 0 AVAX for this ledger)
+
+### Decide on Which Index of Ledger A to Use
+
+It is needed to find out which index is being used by the web wallet. Say that the web wallet
+shows 4.5 AVAX available on p-chain address P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0
+
+With the ledger A connected and the avalache app running, execute:
+
+```bash
+avalanche key list --mainnet --ledger 0,1,2,3,4,5
+```
+
+To see p-chain addresses and balances for the first 6 indices in the ledger derived owner addresses.
+
+```text
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+|  KIND  |  NAME   |          CHAIN          |                    ADDRESS                    | BALANCE | NETWORK |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+| ledger | index 0 | P-Chain (Bech32 format) | P-avax1g8yucm7j0cnwwru4rp5lkzw6dpdxjmc2rfkqs9 |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 1 |                         | P-avax1drppshkst2ccygyq37m2z9e3ex2jhkd2txcm5r |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 2 |                         | P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0 |     4.5 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 3 |                         | P-avax1yfpm7v5y5rej2nu7t2r0ffgrlpfq36je0rc5k6 |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 4 |                         | P-avax17nqvwcqsa8ddgeww8gzmfe932pz2syaj2vyd89 |       0 | Mainnet |
++        +---------+                         +-----------------------------------------------+---------+---------+
+|        | index 5 |                         | P-avax1jzvnd05vsfksrtatm2e3rzu6eux9a287493yf8 |       0 | Mainnet |
++--------+---------+-------------------------+-----------------------------------------------+---------+---------+
+```
+
+As can be noticed, the address `P-avax10an3cucdfqru984pnvv6y0rspvvclz63e523m0` indeed has 4.5 AVAX and
+is associated to index 2 of the ledger A.
+
+:::note
+
+A ledger manages an infinite amount of addresses derived from a main private key, and due to this,
+in certain operations there is a need to specify which address or which index of the ledger is the user
+operating against.
+
+### Decide which Index of Ledger B to Use and Find Out the Associated Address
```

Comment:
These subtitles appear in the sidebar on the right-hand side of the page and are easily truncated when they are too long. 
```suggestion
### Determine Receiver Address Index
```